lib/translator/BUILD

```diff
@@ -1,3 +1,16 @@
+cc_library(
+  name = "read_capacity_10_lib",
```

Comment:
Are we planning on supporting ReadCapacity16?  Any reason to not make a single `read_capacity_lib` and have both 10 and 16 in the same file? (This is the pattern that Read and Write appear to be following)

Reply:
ReadCapacity10 was specifically specified for minimum core functionality by @w-michal. When we add ReadCapacity16, it does make sense to add it to this file.